docs/whirlpool/docs/05-More Resources/04-API.mdx

```diff
@@ -0,0 +1,20 @@
+---
+sidebar-label: API
+---
+import DocCard from '@theme/DocCard';
```

Comment:
Same as with the Rust SDK section. I guess this is fine for now but we should probably make this it's own section in the docs with public api v2? Can you create a ticket for this?